code/API_definitions/location-subscriptions.yaml

```diff
@@ -0,0 +1,534 @@
+          required: true
+          schema:
+            type: string
+      responses:
+        '200':
+          description: Contains information about Event Subscriptions
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/EventSubscriptionInfo'
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '404':
+          $ref: '#/components/responses/Generic404'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503' 
+    delete:
+      tags:
+        - Location Event Subscription
+      summary: 'Operation to delete a event subscription'
+      operationId: deleteEventSubscription
+      description: Delete a given subscription of event by ID
+      parameters:
+        - name: eventSubscriptionsId
+          in: path
+          description: event subscription identifier
+          required: true
+          schema:
+            type: string
+      responses:
+        '204':
+          description: Event subscription deleted
+        '400':
+          $ref: '#/components/responses/Generic400'
+        '401':
+          $ref: '#/components/responses/Generic401'
+        '403':
+          $ref: '#/components/responses/Generic403'
+        '404':
+          $ref: '#/components/responses/Generic404'
+        '500':
+          $ref: '#/components/responses/Generic500'
+        '503':
+          $ref: '#/components/responses/Generic503'
+ 
+components:
+  securitySchemes:
+    oAuth2ClientCredentials:
+      type: oauth2
+      flows:
+        clientCredentials:
+          tokenUrl: '{tokenUrl}'
+          scopes: {}
+    three_legged:
+      type: oauth2
+      flows:
+        authorizationCode:
+          authorizationUrl: https://auth.example.com/authorize
+          tokenUrl: https://auth.example.com/token
+          scopes: {}
+  
+  schemas:
+    Area:
+      oneOf: 
+        -  $ref: "#/components/schemas/Circle"
+      discriminator: 
+        propertyName: type
+        mapping: 
+          Circle:  "#/components/schemas/Circle"
+    
+    Circle:
+      type: object
+      description: Circular area
+      required:
+        - type
+        - location
+        - accuracy
+      properties:
+        type:
+          type: string
+          description:  Type of this area.
+        location:
+          $ref: '#/components/schemas/Point'
+        accuracy:
+          type: number
+          description: Expected accuracy for the subscription event of device location in km, from location (radius)
+          minimum: 2
+          maximum: 200
+      example:
+        type: Circle
+        location:
+          latitude: 50.735851
+          longitude: 7.10066
+        accuracy: 50
+    
+    Point:
```

Comment:
This is not used anywhere

Reply:
Do you Point is not used?! if you check line 211, point is used as a property of Circle object